VMWare/vmware-vcenter/tests/http_logs_2.json

```diff
@@ -0,0 +1,62 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-vcenter",
+        "dialect_uuid": "0642b03a-9d4a-4c88-a5e2-4597e366b8c4"
+      }
+    },
+    "message": "[11/May/2023:10:22:24 +0000] tomcat-http--47 [Request] 1.2.3.4:48866 to local 1080 - HTTP/1.1 POST /lookupservice/sdk [Response] 200 - 758 bytes [Perf] process 0ms / commit 0ms / conn [+]"
+  },
+  "expected": {
+    "message": "[11/May/2023:10:22:24 +0000] tomcat-http--47 [Request] 1.2.3.4:48866 to local 1080 - HTTP/1.1 POST /lookupservice/sdk [Response] 200 - 758 bytes [Perf] process 0ms / commit 0ms / conn [+]",
+    "event": {
+      "kind": "event",
+      "category": [
+        "network"
+      ],
+      "type": [
+        "connection"
+      ]
+    },
+    "@timestamp": "11/May/2023:10:22:24 +0000",
+    "http": {
+      "request": {
+        "method": "POST"
+      },
+      "response": {
+        "status_code": 200,
+        "bytes": 758
+      },
+      "version": "1.1"
+    },
+    "url": {
+      "path": "/lookupservice/sdk"
+    },
+    "process": {
+      "end": "0",
+      "name": "tomcat-http--47 [Request]"
```

Comment:
I think that `tomcat-http--47` is a thread name, and `[Request]` should not be extracted (I think you can add [Request] in the grok pattern).